docs/api_reference.rst

```diff
@@ -35,6 +35,13 @@ Programs
 
         :vartype: Platform or None
 
+    .. attribute:: language
+
+        Primary language that this program uses, or ``None`` if it has not
+        been determined yet.
+
+        :vartype: Language or None
```

Comment:
Documentation needs updating since this can no longer be `None`. (Note that the work I just pushed moved this documentation to `_drgn.pyi`).